### DISCUSSION TITLE
Camera Performance - AI Disabled vs Enabled

### DISCUSSION THREAD

**Discussion by ChaoSandwich** [koush/scrypted#1192] [Q&A] [answered]
In the Codec settings tab, it said '_Some cameras, such as Hikvision or Amcrest, allow more streams and higher fps when AI is disabled. The camera AI is not used within Scrypted, and can be disabled for better performance._'

I have not been able to find confirmation for this statement anywhere else...can anyone confirm if that's really the case and if so what is the improvement per stream? 30fps -> 60? 720p to 1080?

EDIT: As an example I'm looking at the DS-2CD2T87G2-LSU/SL Its listed stream specs are:
     Main Stream
     --25 fps (3840 × 2160, 3200 × 1800 , 2688 × 1520, 1920 × 1080, 1280 × 720)
     --24 fps (3840 × 2160)
     --30 fps (3200 x 1800 , 2688 × 1520, 1920 × 1080, 1280 × 720)
     Sub-Stream
     --25 fps (1280 × 720, 640 × 480, 640 × 360)
     --30 fps (1280 × 720, 640 × 480, 640 × 360)
     Third Stream
     --10 fps (1920 x 1080 , 1280 × 720, 640 × 480, 640 × 360)

Thank you for any and all info you can give

**Comment by koush** [comment, marked as answer]
Yes, that's the case. I have hikvision and amcrest cameras that only expose the third stream when AI is disabled. This restriction only exists on some cameras. I also have cameras that have 3 streams available even with AI enabled.

**Reply by ChaoSandwich** [reply]
Thank you, do you mean the specs that they list on their site are only with AI disabled? Or do they get 1 stream more than their listed specs with ai disabled?
